x-pack/legacy/plugins/siem/common/utility_types.ts

```diff
@@ -4,8 +4,16 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { ReactNode } from 'react';
+
 export type Pick3<T, K1 extends keyof T, K2 extends keyof T[K1], K3 extends keyof T[K1][K2]> = {
   [P1 in K1]: { [P2 in K2]: { [P3 in K3]: ((T[K1])[K2])[P3] } }
 };
 
 export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+// This type is for typing EuiDescriptionList
+export interface DescriptionList {
```

Comment:
Niice! Thanks for adding this type and using it in `HostOverview` and `IpOverview` as well! 🎉